Frontend/ui-common/src/Engineering/Helpers/JBridge.purs

```diff
@@ -206,6 +204,30 @@ foreign import cleverTapSetLocation :: Unit -> Effect Unit
 foreign import saveSuggestions :: String -> Suggestions -> Unit
 foreign import saveSuggestionDefs :: String -> SuggestionDefinitions -> Unit
 foreign import launchDateSettings :: String -> Unit
+foreign import startLottieProcess :: AnimationConfig -> Unit
+foreign import methodArgumentCount :: String -> Int
+
+type AnimationConfig = {
+    rawJson :: String
+  , lottieId :: String
+  , repeat :: Boolean
+  , speed :: Number
+  , scaleType :: String
+  , minProgress :: Number
+  , maxProgress :: Number
+}
+
+animationConfig :: AnimationConfig
+animationConfig = {
+    rawJson : ""
+  , lottieId : ""
+  , repeat : true
+  , speed : 0.6
+  , scaleType : "Default"
```

Comment:
@ashishsingh101 can we send the `Default` in all Caps (as anyhow it goes to the default case).